pkg/sdkserver/localsdk.go

```diff
@@ -384,28 +387,139 @@ func (l *LocalSDKServer) stopReserveTimer() {
 // [Stage:Alpha]
 // [FeatureFlag:PlayerTesting]
 func (l *LocalSDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerID) (*alpha.Bool, error) {
-	panic("implement me")
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return nil, errors.New(string(runtime.FeaturePlayerTracking) + " not enabled")
+	}
+	logrus.WithField("playerID", id.PlayerID).Info("Player Connected")
```

Comment:
```suggestion
	l.logger.WithField("playerID", id.PlayerID).Info("Player Connected")
```

Reply:
Ah very nice, I missed that. Making these changes 👍